aat/src/aat/java/uk/gov/hmcts/ccd/printservice/befta/CasePrintServiceTestAutomationAdapter.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.hmcts.ccd.printservice.befta;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import uk.gov.hmcts.befta.BeftaMain;
+import uk.gov.hmcts.befta.DefaultTestAutomationAdapter;
+import uk.gov.hmcts.befta.data.UserData;
+import uk.gov.hmcts.befta.exception.FunctionalTestException;
+
+public class CasePrintServiceTestAutomationAdapter extends DefaultTestAutomationAdapter {
```

Comment:
The test data we're loading here may change later, but we are good to start like this, subject to later modifications as needed and realized.